Form-Controls/styles.css

```diff
@@ -0,0 +1,14 @@
+#name {
+    margin: 10px;
```

Comment:
Hi @Daniel-Alexander-The-Great ,
I will suggest using rem except if you want the element to have a fixed size e.g (border, and box-shadow).